index.js

```diff
@@ -17,6 +19,21 @@ module.exports = function (str, cb) {
 	});
 };
 
+module.exports.stream = function () {
+	var size = 0;
+	return zlib.createGzip(opts)
+		.on('data', function (buf) {
+			size += buf.length;
+		})
+		.on('error', function () {
+			this.size = 0;
+		})
+		.on('end', function () {
+			this.size = size;
+			this.emit('size', size);
+		});
+};
```

Comment:
Done! I noticed that [ava](https://www.npmjs.com/package/ava) wasn't correctly counting the test plan. I tried to go to the repo but it 404s. I'm guessing you discontinued it? Anyway, I switched to [tap](https://github.com/isaacs/node-tap) – hope that's ok.